### PR TITLE
[documentation] Storybook 5 server not running on React Native

### PR DESCRIPTION
**Pull request by roydipti23**
https://github.com/storybookjs/storybook/issues/6674

Update in documentation

`[ "documentation"]`

### CODE REVIEW

File: docs/src/pages/guides/guide-react-native/index.md [hunk 1]
```diff
@@ -193,7 +193,7 @@ Then:
 {
   "scripts": {
     ...
-    "storybook": "start-storybook -p 6006"
+    "storybook": "adb reverse tcp:7007 tcp:7007 && start-storybook"
```
Comment (shilman) [inline docs/src/pages/guides/guide-react-native/index.md]: Is this for Android only? Does it cause any problems with iOS?

Comment (roydipti23) [reply docs/src/pages/guides/guide-react-native/index.md]: HI, i do not have a ios setup to check with. Will get back to you once i do.

Comment (Gongreg) [inline docs/src/pages/guides/guide-react-native/index.md]:
```suggestion
    "storybook": "(adb reverse tcp:7007 tcp:7007 || true) && start-storybook"
```

**Review by Gongreg — APPROVED** [review]

**Comment by Gongreg** [comment]
@shilman, lets merge it?

**Pull request merged**